templates/sample/src/csblocks.css

```diff
@@ -0,0 +1,30 @@
+body {
```

Comment:
this file should be incorporated into the `scss` structure we have in `static/scss`.

Reply:
I moved this file to the folder static/css accordingly.